examples/gpu/kubernetesgpu.json

```diff
@@ -0,0 +1,35 @@
+{
+  "apiVersion": "vlabs",
```

Comment:
Since GPU is enabled by default and ACS Engine API doesn't provide a way to disable GPU, is a separate example needed? Since this example doesn't do anything new compared to other examples. It's better to update documentation with the info that GPU is enabled by default and will work on GPU enabled skus.